manifests/gcp_marketplace/schema.yaml

```diff
@@ -184,8 +184,8 @@ properties:
     description: |-
       If you are deploying Kubeflow Pipelines with managed storage, specify the
       database name prefix. The prefix value must contain only lowercase
-      letters, numbers, and hyphens, and the prefix must start with a letter.
```

Comment:
so hyphens not supported? thanks for the fix!

Reply:
reading the internal bug, I think we should also mention underscore is also allowed

Reply:
Not seeing any response, but I want to cherry pick this to release branch, edited in your PR directly. Please take a look when you are back.

Reply:
OK with the change.